nucleus/dataset.py

```diff
@@ -438,13 +438,15 @@ def create_model_run(
         model_id: Optional[str] = None,
         metadata: Optional[Dict[str, Any]] = None,
         annotation_metadata_schema: Optional[Dict] = None,
+        trained_slice_ids: Optional[str] = None,
```

Comment:
does it still make sense to add this to `create_model_run` now that trained_slice_id is associated to the modelproject object?

Reply:
This was not meant to be pushed, however, with it being added to model projects object I do not see when we would ever use it with model runs. So I guess that also begs the question of if we should revert the changes for adding it to model run or just leave it in case of future uses.

Reply:
I'd go with reverting the changes which added it to model_run to eliminate the confusion

Reply:
Just to clarify (since I reread what I sent and it was not super clear). Definitely remove it here, but I was referring to also reverting the changes in https://github.com/scaleapi/nucleus-python-client/pull/417 or should we leave those around?

Reply:
Yes I was referring to reverting the changes in https://github.com/scaleapi/nucleus-python-client/pull/417, otherwise it would be really confusing for the user to decide which api to choose.

Reply:
Will create a separate PR to pull out #417 changes.